pyproject.toml

```diff
@@ -7,32 +7,25 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">=3.10,<3.12"
-adlfs = "*"
-azure-identity = "*"
 cartopy = "*"
-eccodes = "*"
-fastparquet = "*"
-fsspec = "*"
+dask = {version = "*", extras = ["complete"]}
 geopandas = "*"
-jupyter-bokeh = "*"
-lttb = "*"
 matplotlib = "*"
 observer = {git = "https://github.com/seareport/observer.git"}
 owslib = "*"
 pandas = "*"
-pyarrow = "*"
 pyposeidon = {git = "https://github.com/ec-jrc/pyPoseidon.git"}
 ruptures = "*"
-scikit-image = "*"
```

Comment:
this require to also remove https://github.com/seareport/analysea/blob/f3f863a9424ea7ae17d75f50443c50fbcbe1044c/analysea/steps.py#L12